cmd/gomd/commands.go

```diff
@@ -127,3 +133,10 @@ func (a *app) cmdMkdir() error {
 func (a * app) cmdSync() error {
 	return a.right.Folder.SetDir(a.left.Folder.Path)
 }
+
+func (a * app) cmdSwitch() error {
+	leftPath := a.left.Folder.Path
+	rightPath := a.right.Folder.Path
+	a.left.Folder.SetDir(rightPath)
```

Comment:
If we return the error of the right side, we should handle this error here, too.

Reply:
Okay. If we handle both the errors separately, it can work.

Reply:
I have created a new commit which handles error from both the function calls.